istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -1480,7 +1391,6 @@ func ProxyConfig(ctx cli.Context) *cobra.Command {
 	configCmd.AddCommand(secretConfigCmd(ctx))
 	configCmd.AddCommand(rootCACompareConfigCmd(ctx))
 	configCmd.AddCommand(ecdsConfigCmd(ctx))
-	configCmd.AddCommand(workloadConfigCmd(ctx))
```

Comment:
Since the command was unstable and hidden from users, I think there's no compatibility issue to worry, and we can directly add it to the experimental commands?